package lab4;

import java.io.IOException;

public class Program01 {
	public void method() {
		// throw new IOException(); shows error since it must be surrounded by try catch
	}
	public static void main(String[] args) {
		
	}
}
